forge-gui/res/cardsfolder/upcoming/razorfield_ripper.txt

```diff
@@ -0,0 +1,11 @@
+Name:Razorfield Ripper 
+ManaCost:2 W
+Types:Artifact Creature Equipment Rhino
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self,Creature.EquippedBy | Execute$ TrigPutCounter | TriggerDescription$ Whenever CARDNAME or equipped creature attacks, you get {E}, then it gets +X/+X until end of turn, where X is the amount of {E} you have.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 1 | SubAbility$ DBPump
+SVar:DBPump:DB$ Pump | Defined$ TriggeredAttackerLKICopy | NumAtt$ +X | NumDef$ +X
+K:Reconfigure:2:::AlternateCost$ PayEnergy<3>
```

Comment:
did you test this?

Reply:
Hmm I didn't, I used Bloodthorn Flail's script as an example.  I guess this needs to be wired up first ?

Reply:
I will see what can be done